### PR TITLE
Fix a couple of About Screen issues

### PR DESCRIPTION
**Pull request by frosty**
This PR fixes a couple of small issues with the new About screen.

1. On some devices (only on device, and it doesn't show up on screenshots), a faint shimmering outline was visible around the app logos view:

I have only seen this on my 13 mini. This is now fixed by setting `allowsTransparency` to `true` on the SpriteKit view.

2. The text in the header view didn't respond to dynamic type changes, so I've improved that. There's still an issue for the largest accessibility text size where a very large version number can get slightly truncated, but I spent a little while on it and it's much improved from what we had before.

| Before | After |
|---|---|
| Simulator Screen Shot - iPhone 13 Pro - 2021-12-08 at 15 46 37 | Simulator Screen Shot - iPhone 13 Pro - 2021-12-08 at 14 38 07 |

**To test**

* Build and run
* Navigate to Me > About
* Check the header looks as you'd expect, based on the Before screenshot above
* Try changing the dynamic font sizes and ensure it looks okay
* If you toggle it while the screen is visible, occasionally there may be a spacing issue below the header but it'll resolve itself if you reshow the view or change between other sizes.

## Regression Notes

1. Potential unintended areas of impact

None

2. What I did to test those areas of impact (or what existing automated tests I relied on)
3. What automated tests I added (or what prevented me from doing so)

N/A

PR submission checklist:

- [x] I have completed the Regression Notes.
- [x] I have considered adding unit tests for my changes.
- [x] I have considered adding accessibility improvements for my changes.
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

cc @mokagio

### CODE REVIEW

File: WordPress/Classes/ViewRelated/Me/App Settings/About/AutomatticAboutScreen.swift [hunk 1]
```diff
@@ -62,7 +62,7 @@ class AutomatticAboutScreen: UIViewController {
 
         // Setting the frame once is needed so that the table view header will show.
         // This seems to be a table view bug although I'm not entirely sure.
-        headerView.frame.size = headerView.systemLayoutSizeFitting(UIView.layoutFittingCompressedSize)
+        headerView.frame.size.height = headerView.systemLayoutSizeFitting(UIView.layoutFittingCompressedSize).height
```
Comment (Gio2018) [inline WordPress/Classes/ViewRelated/Me/App Settings/About/AutomatticAboutScreen.swift:65]: I agree that this feels a bit like a hack, but if that's the only way for the header to show..

**Review by Gio2018 — APPROVED** [review]
I did not have an iPhone 13 mini device, tested on iPhone 12 Pro Max and worked as expected.

**Comment by mokagio** [comment]
@frosty   this has been bundled as part of 18.8 beta 4 (18.8.0.4), which is currently building in CI.

Thanks for your work 🙌

**Comment by frosty** [comment]
Thank _you_!

**Pull request merged**